CONTRIBUTING.md

```diff
@@ -9,13 +9,27 @@ Both bug reports and feature requests are welcome.
 Submit issues [here](https://github.com/Kotlin/kotlinx.coroutines/issues).
 Questions about usage and general inquiries are better suited for [StackOverflow](https://stackoverflow.com)
 or the `#coroutines` channel in [KotlinLang Slack](https://surveys.jetbrains.com/s3/kotlin-slack-sign-up).
+There, questions get answered much quicker than they do in this issue tracker,
+while also reducing the load on the maintainers.
+
+### Commenting on issues
+
+Describing why you're interested in some specific issue helps us a lot with looking for the best solution.
+The more experiences, concerns, or suggestions get shared, the better!
+If you feel that something you know hasn't been said (or even if the issue discussion is too long to actually
+make sure of it), please do not hesitate to drop a line.
+
+On the other hand, "+1"-style remarks or questions about the progress on an issue do not bring anything new
```

Comment:
Although, 👍 vote on an issue is an indicator of community's interest in a fix. So it might worth suggesting to vote by adding a reaction, instead of writing "me too" or "+1" comments.